website/static/js/osfHelpers.js

```diff
@@ -838,6 +838,17 @@ var findContribName = function (userAttributes) {
     }
 };
 
+// Check if a user is in the list of project contributors returned by the v2 API.
+function userIsContributor(userId, contributors) {
```

Comment:
There's nothing really specific about contributors about this function; I wonder if it can be generalized.

How about implementing a simplified version of [_.find](https://lodash.com/docs#find)? 

> Iterates over elements of collection, returning the first element predicate returns truthy for.

I also wouldn't be opposed to adding `lodash.find` as a dependency.